src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb

```diff
@@ -1224,7 +1226,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
             End If
 
-            Return New BoundLocalDeclaration(name, symbol, valueExpression, isInitializedByAsNew)
+            Return New BoundLocalDeclaration(name, symbol, valueExpression, isInitializedByAsNew, boundArrayCreation)
```

Comment:
>Return New BoundLocalDeclaration(name, symbol, valueExpression, isInitializedByAsNew, boundArrayCreation) [](start = 12, length = 105)

Is it possible that ```valueExpression``` and ```boundArrayCreation``` refer to the same node in some scenarios?

Reply:
Yes, it is possible. I'm taking advantage of that in the vb operation factory.

---
In reply to: [146950870](https://github.com/dotnet/roslyn/pull/22842#discussion_r146950870) [](ancestors = 146950870)